templates/admin-footer.html

```diff
@@ -297,15 +297,23 @@
             }
         });
 
-        $('#search_form input[type="text"]').keyup(function(){
+        $('#search_form input[type="text"]').keyup(function(event){
             event.preventDefault();
 
             let _input = $('#search_form input[type="text"]'),
+                _filter = _input.val().toLowerCase(),
                 _btn = $('#search_form .btn');
 
-            if (_input.val().length > 2) {
+            if (_input.val().length > 0) {
                 _btn.removeClass('disabled').removeAttr('disabled');
+                $('#sites>li').each(function(){
+                     let _title = $(this).data('title').toLowerCase();                
+                    (_title.indexOf(_filter) == 0) ? $(this).show() : $(this).hide();    
```

Comment:
Change == to >= (include instead of start swith)

```
_title = 'edn4501w,2021'
_filter = 'edn';

_title.indexOf(_filter) == 0
true

_filter = '2021';

_title.indexOf(_filter) == 0
false
_title.indexOf(_filter) >= 0
true
```